Cache modules

tmiland opened the pull request:
- [redis2-nginx-module](https://github.com/openresty/redis2-nginx-module) : Nginx upstream module for the Redis 2.0 protocol
- [ngx_http_redis](https://www.nginx.com/resources/wiki/modules/redis/) : The nginx HTTP redis module for caching with redis
- [srcache-nginx-module](https://github.com/openresty/srcache-nginx-module) : Transparent subrequest-based caching layout for arbitrary nginx locations
- [set-misc-nginx-module](https://github.com/openresty/set-misc-nginx-module) : Various set_xxx directives added to nginx's rewrite module (md5/sha1, sql/json quoting, and many more)
- [echo-nginx-module](https://github.com/openresty/echo-nginx-module) : Brings "echo", "sleep", "time", "exec" and more shell-style goodies to Nginx config file.
  - Required to set up [Redis with conditional purging](https://easyengine.io/wordpress-nginx/tutorials/single-site/redis_cache-with-conditional-purging/)
  - Install Redis with ```apt install redis-{tools,server}```

PS: Removed duplicate ```SUBFILTER=${SUBFILTER:-n}```

Installed with:

```
HEADLESS=y \
NGINX_VER=MAINLINE \
PAGESPEED=y \
BROTLI=y \
HEADERMOD=y \
FANCYINDEX=y \
CACHEPURGE=y \
SUBFILTER=y \
LUA=n \
WEBDAV=y \
VTS=y \
RTMP=y \
TESTCOOKIE=y \
REDIS2=y \
HTTPREDIS=y \
SRCACHE=y \
SETMISC=y \
NGXECHO=y \
./nginx-autoinstall.sh 2>&1 | tee nginx-installer.log
```

NOTE: ```LUA=n```
Had issues with ```LUA=y```

Confirm with: 
```
 $ nginx -V 2>&1 | grep 'srcache-nginx-module\|redis2-nginx-module\|ngx_http_redis\|set-misc-nginx-module\|echo-nginx-module' -o

redis2-nginx-module
ngx_http_redis
srcache-nginx-module
set-misc-nginx-module
echo-nginx-module

```

Tested on Debian 10/11.

angristan reviewed (changes requested):
Thanks for the fantastic PR! Could you update `.github/workflows/docker/wrapper.sh` so that the CI can run with the new modules?

tmiland commented:
> Thanks for the fantastic PR! Could you update `.github/workflows/docker/wrapper.sh` so that the CI can run with the new modules?

My pleaseure! 

Done. 
- Updated wrapper.sh for Cache modules
- Removed duplicates (RTMP,SUBFILTER)

tmiland reviewed (commented):
Done.

- Updated wrapper.sh for Cache modules
- Removed duplicates (RTMP,SUBFILTER)
